tensorboard/plugins/interactive_inference/witwidget/notebook/visualization.py

```diff
@@ -580,12 +589,17 @@ def set_ai_platform_model(
       self.store('adjust_example', adjust_example)
     if adjust_attribution:
       self.store('adjust_attribution', adjust_attribution)
+    if get_explanations:
```

Comment:
I think its better if we always store get_explanations so that should_explain should not rely on None vs True for comparison (False vs True is better).

Reply:
agree, done